composer.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "websitetoolbox/websitetoolboxcommunity",
```

Comment:
If the namespace already contains websitetoolbox/, then I don't understand why you didn't just make the namespace websitetoolbox/community and websitetoolbox/community.

Reply:
The namespace was already created previously. So, I keep the name of namespace as it is and not change. 
Should I change it?